src/apps/logging/loggingjs.lua

```diff
@@ -0,0 +1,38 @@
+-- Copyright (c) Microsoft Corporation. All rights reserved.
+-- Licensed under the Apache 2.0 License.
+
+return {
```

Comment:
Should this file have the `.lua` extension?

Reply:
Yes, it's lua! See point 3. in PR description, this lets us re-use existing infrastructure for now.